src/Microsoft.TestPlatform.CommunicationUtilities/Messages/DiscoveryCompletePayload.cs

```diff
@@ -4,31 +4,61 @@
 namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.ObjectModel
 {
     using System.Collections.Generic;
+    using System.Runtime.Serialization;
     using Microsoft.VisualStudio.TestPlatform.ObjectModel;
 
     /// <summary>
     /// The discovery complete payload.
     /// </summary>
     public class DiscoveryCompletePayload
     {
+        [DataMember]
```

Comment:
Why weren't these required earlier?

Reply:
I am not sure, it works fine now, but we decided to add it as "good to have", because payload is used in serialization.